src/ppx/grfx/vk/vk_command.cpp

```diff
@@ -335,18 +362,44 @@ void CommandBuffer::ClearDepthStencil(
     const grfx::DepthStencilClearValue& clearValue,
     uint32_t                            clearFlags)
 {
-    auto pCurrentRenderPass = GetCurrentRenderPass();
-    if (IsNull(pCurrentRenderPass)) {
+    if (!HasActiveRenderPass()) {
         return;
     }
 
-    // Make sure pImage is depth stencil in current render pass
-    if (pCurrentRenderPass->GetDepthStencilImage().Get() != pImage) {
-        return;
+    grfx::Rect renderArea;
+    uint32_t   baseArrayLayer;
+
+    // Dynamic render pass
+    if (mDynamicRenderPassActive) {
+        // Clear rect
+        renderArea = mRenderArea;
+
+        // Make sure pImage is depth stencil in depth stencil view
+        if (mDepthStencilView->GetImage().Get() != pImage) {
```

Comment:
Not sure I follow, in both cases, `pImage` is compared to the attribute, and if unequal, nothing is done. Is this parameter useful? What is the scenario for that?

Reply:
I think it probably should do an assert if the image is not found. If someone passes an image that is not actually a DS attachment, the clear won't work and we probably should inform user about that

Reply:
I've left it in and added the assert if image not found. If you prefer I remove this check altogether - let me know

Reply:
Removed parameter check for both dynamic and non dynamic renderpasses